wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,22 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+This frame limiter attempts to keep a frame rate equal to your custom preference. However, this is a setting that cannot be changed to within the in-game menu.
+
+To change osu! custom maximum framerate, you'll need to tweak some values in the config file. Here's the steps:
+
+- Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Roaming\osu!`.)
+- Open the `osu!.<yourPCusername>.cfg` file with Notepad or any text editors.
+- Find the line `FrameSync = <value>` and change the value to `Custom`.
+- Find the line `CustomFrameLimit = <value>` and change the value to your preference.
+
+This is our recommended frame limiter for users who:
+
+- Want to set the frame limiter at 4x refresh rate, etc. but the game doesn't have the option to do so.
+- Doesn't satisfied with the maximum of 960fps that Optimal gives (but **do not** want to run their CPU/GPU to extremities), resulting in even lower input latency than Optimal without abusing your CPU/GPU too much with Unlimited.
```

Comment:
this line isn't necessary imo; if users want to change it they know why they want to change it. Either way, colon is being used after an incomplete clause

Reply:
I've removed it. Thanks for suggesting.